perfkitbenchmarker/benchmark_spec.py

```diff
@@ -387,8 +393,10 @@ def DeleteVm(self, vm):
 
   def PickleSpec(self):
     """Pickles the spec so that it can be unpickled on a subsequent run."""
+    flags, self.FLAGS = self.FLAGS, None
```

Comment:
Pickle blows up if you try to pickle FlagValues:
PicklingError: Can't pickle <function Checker at 0x7fb410743848>: it's not found as gflags.Checker
Even dill blows up when unpickling:
KeyError: '__flags'